src/meta/processors/job/JobManager.cpp

```diff
@@ -232,8 +231,9 @@ nebula::cpp2::ErrorCode JobManager::jobFinished(JobID jobId, cpp2::JobStatus job
 
   auto it = runningJobs_.find(jobId);
   if (it == runningJobs_.end()) {
-    LOG(INFO) << folly::sformat("can't find jobExecutor, jobId={}", jobId);
-    return nebula::cpp2::ErrorCode::E_UNKNOWN;
+    // the job has not started yet
```

Comment:
If here `jobId`  is not found from `runningJobs_`, JobExecutor should be created.

For example, meta leader change, reportTaskFinish is sent to the new meta leader, and the runningJobs_ in the new meta leader is empty.

Reply:
that's a problem

Reply:
If you can't find it here, create a new JobExecutor

Reply:
yes, but only for storage jobs